docs/source/common_issues.rst

```diff
@@ -808,3 +808,78 @@ not necessary:
     class NarrowerArgument(A):
         def test(self, t: List[int]) -> Sequence[str]:  # type: ignore[override]
             ...
+
+TypeVars are not the same
+------------------------------------------------
+
+As discussed in the issue
+`TypeVars should be allowed to be the same <https://github.com/python/mypy/issues/7864>`_,
+TypeVars are *not* equivalent even if they are identically defined.
```

Comment:
I'd add one or two sentences about why this is the case. I'm not sure what's the best way express this, but each type variable can vary independently.

Reply:
Hi, I'm not sure whether it is valid to say like this. 
`TypeVar' is not one of the types such as `int`, `str`. Hence each type variable is unique even though they have identical signatures since it is not meant to be compared with each other.